src/Microsoft.TemplateEngine.Utils/PublicAPI.Unshipped.txt

```diff
@@ -1,4 +1,12 @@
-﻿static Microsoft.TemplateEngine.Utils.WellKnownSearchFilters.AuthorFilter(string? author) -> System.Func<Microsoft.TemplateEngine.Abstractions.ITemplateInfo!, Microsoft.TemplateEngine.Abstractions.TemplateFiltering.MatchInfo?>!
+﻿Microsoft.TemplateEngine.Utils.CryptoRandom
+Microsoft.TemplateEngine.Utils.CryptoRandom.CryptoRandom() -> void
+Microsoft.TemplateEngine.Utils.CryptoRandom.CryptoRandom(int ignoredSeed) -> void
+override Microsoft.TemplateEngine.Utils.CryptoRandom.Next() -> int
+override Microsoft.TemplateEngine.Utils.CryptoRandom.Next(int maxValue) -> int
+override Microsoft.TemplateEngine.Utils.CryptoRandom.Next(int minValue, int maxValue) -> int
+override Microsoft.TemplateEngine.Utils.CryptoRandom.NextDouble() -> double
+static Microsoft.TemplateEngine.Utils.WellKnownSearchFilters.AuthorFilter(string? author) -> System.Func<Microsoft.TemplateEngine.Abstractions.ITemplateInfo!, Microsoft.TemplateEngine.Abstractions.TemplateFiltering.MatchInfo?>!
```

Comment:
This Author filter thing doesn't seem to be a part of this PR. Is this a leftover API change from a previous PR?
If yes, we may want to check how our CI allowed merging it.

Reply:
Thank you for good catch.
I accidentally didn't properly shelve my local changes (it wasn't part of change that was already pushed)

Reply:
Actually it was already there, only BOM char on beginning of the line disappeared (as it moved from the first line) making GH show it as line deleted and line added.
Other than BOM the line was not touched